alfred-placeholder-workflows.rb

```diff
@@ -1,5 +1,5 @@
 class AlfredPlaceholderWorkflows < Formula
-  desc "Create dummy Alfred Workflows that update to their real counterparts, to host on Packal"
+  desc "Dummy Alfred Workflows for updating real counterparts hosted on Packal"
```

Comment:
This changes the meaning to the opposite.

These also require more changes than just the formulae; these new descriptions have to be updated on the scripts themselves and their respective repo’s READMEs. And I’m not a fan of “&” for replacing a regular “and” in a sentence.

I’ll be fixing all this shortly, thank you for pointing them out, that helped.

Reply:
Done.